terraform/azure/projects/utoronto.tfvars

```diff
@@ -3,8 +3,8 @@ resourcegroup_name = "2i2c-utoronto-cluster"
 
 ssh_pub_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQJ4h39UYNi1wybxAH+jCFkNK2aqRcuhDkQSMx0Hak5xkbt3KnT3cOwAgUP1Vt/SjhltSTuxpOHxiAKCRnjwRk60SxKhUNzPHih2nkfYTmBBjmLfdepDPSke/E0VWvTDIEXz/L8vW8aI0QGPXnXyqzEDO9+U1buheBlxB0diFAD3vEp2SqBOw+z7UgrGxXPdP+2b3AV+X6sOtd6uSzpV8Qvdh+QAkd4r7h9JrkFvkrUzNFAGMjlTb0Lz7qAlo4ynjEwzVN2I1i7cVDKgsGz9ZG/8yZfXXx+INr9jYtYogNZ63ajKR/dfjNPovydhuz5zQvQyxpokJNsTqt1CiWEUNj georgiana@georgiana"
 
-global_container_registry_name = "2i2cUTorontoHubRegistry"
-global_storage_account_name = "2i2cUTorontoHubStorage"
+global_container_registry_name = "2i2cutorontohubregistry"
```

Comment:
@GeorgianaElena I had thought you were suggesting using naming like `2i2c-utoronto-hub-registry` - I feel like that is much easier to read than having it all lumped together and lowercase, but maybe I was mis-interpreting your suggestion?

Reply:
Azure is super strict on naming conventions for some resources, that also have to be unique _across all of Azure_. I don't think these resources allow hyphens. https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules

Reply:
god that's infuriating haha, OK then lowercase is fine with me